dotCMS/src/main/java/com/dotmarketing/portlets/containers/action/EditContainerAction.java

```diff
@@ -559,43 +563,77 @@ public void _saveWebAsset(ActionRequest req, ActionResponse res,
 		identifier.setHostId(host.getIdentifier());
 		APILocator.getIdentifierAPI().save(identifier);
 
+
+		//Save/Update/Delete the ContainerStructures associated
+		saveContainerStructures(req, currentContainer, container);
+
+
+		SessionMessages.add(httpReq, "message", "message.containers.save");
+		ActivityLogger.logInfo(this.getClass(), "Save WebAsset action", "User " + user.getPrimaryKey() + " saved " + container.getTitle(), HostUtil.hostNameUtil(req, _getUser(req)));
+		// saves to working folder under velocity
+		ContainerServices.invalidate(container, true);
+
+		// copies the information back into the form bean
+		BeanUtils.copyProperties(form, req
+				.getAttribute(WebKeys.CONTAINER_FORM_EDIT));
+
+		APILocator.getVersionableAPI().setWorking(container);
+
+
+
+
+		HibernateUtil.flush();
+	}
+
+	/**
+	 * Containers are associated with multiple Structures. This method takes care of the ContainerStructure save/update/delete logic
+	 * @param req Request
+	 * @param currentContainer Container as currently exists
+	 * @param container Container with the new changes to be persisted
+	 */
+	private void saveContainerStructures (ActionRequest req, Container currentContainer, Container container) throws DotDataException, DotSecurityException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 154 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)